libr/config/config.c

```diff
@@ -83,39 +179,33 @@ R_API void r_config_list(RConfig *cfg, const char *str, int rad) {
 			cfg->cb_printf ("%s\n", node->name);
 		}
 		break;
+	case 'J':
+		verbose = true;
 	case 'j':
-		cfg->cb_printf ("{");
+		json = true;
+		json_is_first = true;
```

Comment:
last coding style rules says that static and locals should use camelcase, only public/exposed stuff uses the snake style